### PR TITLE
Update youtube-channels.csv

### PR DESCRIPTION
**Pull request by PeppePasti**
Added many missing channels

### CODE REVIEW

File: youtube-channels.csv [hunk 1]
```diff
@@ -546,3 +546,37 @@ UCb4XH3v5SKMQXJWL_lAiUbg, beatsolow, https://www.youtube.com/channel/UCb4XH3v5SK
...
+FrenchcoreHardcore, Frenchcore Hardcore, https://www.youtube.com/c/FrenchcoreHardcore
+@ourgrnd, OURGRND, https://www.youtube.com/@ourgrnd
+TrueHardtranceSounds, TrueHardtranceSounds, https://www.youtube.com/user/TrueHardtranceSounds
+TrueHardstyleSounds, TrueHardstyleSounds, https://www.youtube.com/user/TrueHardstyleSounds
+TheLegendaryUploadzz, TheLegendaryUploadzz, https://www.youtube.com/c/TheLegendaryUploadzz
+TheHardstyleMusicz, TheHardstyleMusicz, https://www.youtube.com/c/TheHardstyleMusicz
+Davide1437, HOUSE ALL TIMES, https://www.youtube.com/user/Davide1437
+nareku, Nareku, https://www.youtube.com/c/nareku
+UCmz8lDqD2RvhSirXNsz_raw, UNSR & DEMENTORS UNITY, https://www.youtube.com/channel/UCmz8lDqD2RvhSirXNsz_raw
+RawhausTrax, RawhausTrax / David Mast, https://www.youtube.com/user/RawhausTrax
```
Comment (stephanebruckert) [inline youtube-channels.csv:582]: rows need to start with the channel ID
also I see there are a lot of duplicates

Comment (PeppePasti) [reply youtube-channels.csv:582]: how do i get the channel id? Cause youtube doesn't show them anymore

Comment (stephanebruckert) [reply youtube-channels.csv:582]: use https://commentpicker.com/youtube-channel-id.php

Comment (PeppePasti) [reply youtube-channels.csv:582]: thanks

**Pull request closed without merging**